lib/std/os/uefi/protocols/edid_override_protocol.zig

```diff
@@ -31,8 +29,8 @@ pub const EdidOverrideProtocol = extern struct {
     };
 };
 
-pub const EdidOverrideProtocolAttributes = packed struct {
+pub const EdidOverrideProtocolAttributes = packed struct(u32) {
     dont_override: bool,
     enable_hot_plug: bool,
-    _pad: u30 = 0,
+    _pad: u30,
```

Comment:
Part of these changes I merged in from another branch, I had removed them because most of the structs come from uefi. I can see how it would help with initializing them though.